inst/maintenance/unit/test-simeta.R

```diff
@@ -209,6 +209,23 @@ test_that("warn when simeps(n) is called with off diagonals", {
   expect_silent(mcode("simeps-n-nowarn-2", code, compile = FALSE))
 })
 
+# Fixes issue discovered in #1092
+test_that("etasrc works with ETA in first column", {
+  mod <- param(mod, mode = 0)
+  
+  data <- expand.ev(amt = 100, ID = seq(4), cmt = 1)
+  data <- dplyr::mutate(data, ETA1 = rev(ID) / 10)
+  data <- expand_observations(data, times = seq(5))
+  data <- dplyr::mutate(data, cmt = 0)
+  
+  set.seed(9812)
+  expect_identical(
+    mrgsim(mod, data, etasrc = "data"),
+    mrgsim(mod, dplyr::select(data, "ETA1", tidyselect::everything()), 
```

Comment:
Interesting that the R 3.6 (but not R 4.0 or R 4.1) drone build [failed](https://github-drone.metrumrg.com/metrumresearchgroup/mrgsolve/1494/1/5) without f0354c3a given the pre-existing `library(mrgsolve)` at the top of the file.